javacutil/src/org/checkerframework/javacutil/TypesUtils.java

```diff
@@ -132,9 +133,24 @@ public static boolean isThrowable(TypeMirror type) {
     }
 
     /**
+     * Returns true iff the argument is an anonymous type.
+     *
+     * @return whether the argument is an anonymous type
+     */
+    public static boolean isAnonymous(TypeMirror type) {
```

Comment:
In a comment, `checker/src/org/checkerframework/checker/guieffect/GuiEffectTypeFactory.java` refers to `TypesUtils.isAnonymousType`, but such a method doesn't exist.
`InternalUtils.java` has `isAnonymousConstructor(final MethodTree method)` but a cross-reference doesn't seem necessary.
I didn't find any other method like this, so I think this is a good addition.